README.md

```diff
@@ -79,13 +83,13 @@ extern crate alloc;
 
 to the preamble and store your example in the `examples-alloc` folder.
 
-To run on the code on your board you can use
+To build the examples for your board you can use
 
 ```shell
 make <platform> [FEATURES=alloc]
 ```
 
-The example can also be flashed to the board by running:
+A program can be flashed to your board after the build process by running:
```

Comment:
I think "An example" would be better than "A program" here.

Reply:
Agreed